base/float.jl

```diff
@@ -432,15 +429,15 @@ function unbiased_exponent(x::T) where {T}
 end
 
 function explicit_mantissa_noinfnan(x::T) where {T}
-    subnormal = unbiased_exponent(x) > uinttype(T)(0) ? significand_mask(T) + uinttype(T)(1) : uinttype(T)(0)
+    subnormal = issubnormal(x) ? significand_mask(T) + uinttype(T)(1) : uinttype(T)(0)
     return subnormal | (significand_mask(T) & uintbits(x))
 end
 
-function make_value(number::T, ep) where {T}
+function make_value(number::T, ep) where T<:Unsigned
     Tfloat = floattype(T)
-    Tint = inttype(T)
+    Tint = signed(T)
     epint = unsafe_trunc(Tint,ep)
-    lz::inttype(T) = Core.Intrinsics.ctlz_int(number) - exponent_width(Tfloat)
+    lz::signed(T) = Core.Intrinsics.ctlz_int(number) - exponent_width(Tfloat)
```

Comment:
`::signed(T)` doing anything?

Reply:
Lemme check, i added it for some reason but maybe that got fixed.